resource/StandardResources.md

```diff
@@ -0,0 +1,94 @@
+# Standard Resources
+
+This page lists the standard resource types in OpenCensus. For more details on how resources can 
+be combined see [this](Resource.md).
+
+OpenCensus defines these fields.
+ * [Compute Unit](#compute-unit)
+   * [Container](#container)
+ * [Deployment Service](#deployment-service)
+   * [Kubernetes](#kubernetes)
+ * [Compute Instance](#compute-instance)
+   * [Host](#host)
+ * [Environment](#environment)
+   * [Cloud](#cloud)
+   * [Cluster](#cluster)
+
+## TODOs
+* Add logical compute units: Service, Task - instance running in a service.
```

Comment:
one more open items for me is a type of values (in this spec- mostly strings). For strings we can also spec a limit on string length (it will help consistency and protect from misuse)

Reply:
We defined limits here: 
https://github.com/census-instrumentation/opencensus-specs/blob/master/resource/Resource.md#resource-type